crates/isograph_schema/src/validate_schema.rs

```diff
@@ -564,8 +586,16 @@ fn validate_use_refetch_field_strategy(
         vec![],
         client_field_name,
     )
-    .map_err(|err| {
-        validate_selections_error_to_validate_schema_error(err, parent_object, client_field_name)
+    .map_err(|errs| {
+        errs.into_iter()
+            .map(|err| {
+                validate_selections_error_to_validate_schema_error(
+                    err,
+                    parent_object,
+                    client_field_name,
+                )
+            })
+            .collect::<Vec<_>>()
```

Comment:
I don't really see how to avoid collecting here, though, without specifying some ugly type